src/ResourceManager/Network/Commands.Network/help/Get-AzureRmDdosProtectionPlan.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version:
```

Comment:
@murilogr please refer to the `ValidateHelpIssues.csv` file that is a part of your Jenkins build for information about how to resolve the build errors around your help files